packages/xo-server-sdn-controller/src/index.js

```diff
@@ -0,0 +1,810 @@
+import assert from 'assert'
+import createLogger from '@xen-orchestra/log'
+import NodeOpenssl from 'node-openssl-cert'
+import { EventEmitter } from 'events'
+import { existsSync, readFileSync, writeFile } from 'fs'
```

Comment:
Never use sync methods because they will block the whole program (`xo-server`), use their async equivalents.

To make them work with `await` you can either use `promisify` or `fromCallback`.